app/models/work.rb

```diff
@@ -487,6 +496,17 @@ def attach_s3_resources
     log_file_changes(changes, nil)
   end
 
+  # Returns true if the work matches the search terms. It searches inside
+  # the resource, the curator, and the work fields.
+  # Uses a brute-force approach to search for the value.
+  # A more fine tuned version could be developed to search within certain specific fields.
+  def match?(search_terms)
+    return true if resource && resource.to_json.downcase.include?(search_terms)
+    return true if curator && curator.to_json.downcase.include?(search_terms)
+    return true if to_json.downcase.include?(search_terms)
+    false
```

Comment:
Slight preference to just make this one big boolean.

Reply:
Method is now gone.